yoti-sdk-api/src/main/java/com/yoti/api/client/docs/session/retrieve/ResourceContainer.java

```diff
@@ -0,0 +1,44 @@
+package com.yoti.api.client.docs.session.retrieve;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class ResourceContainer {
+
+    @JsonProperty("id_documents")
+    private List<DocumentResourceResponse> idDocuments;
+
+    @JsonProperty("liveness_capture")
+    private List<LivenessResourceResponse> livenessCapture;
+
+    @JsonProperty("checks")
+    private List<CheckResponse> checks;
```

Comment:
Based on the documentation `checks` should only be on `DocScanSession` (not in `ResourceContainer`)

Reply:
Fixed in the latest draft